pom.xml

```diff
@@ -605,6 +605,10 @@
                     <argLine>
                         --illegal-access=permit
                     </argLine>
+                    <excludes>
+                        <!--run DAO tests manually -->
```

Comment:
Probably we should come up with a better filter for the unit tests like that sometime later. But this is good for now.

Reply:
Yup, Agreed